vital/types/image.h

```diff
@@ -537,6 +537,15 @@ class VITAL_EXPORT image
    */
   void set_size( size_t width, size_t height, size_t depth );
 
+  /// Get a cropped view of the image
+  /**
+   * Get a cropped view of the image.
```

Comment:
It may be worth being extra explicit here in the description that the cropped image returned shares memory with the source image and is not a deep copy.